src/modules/Dropdown/Dropdown.jsx

```diff
@@ -32,10 +54,15 @@ export default {
       type: Boolean,
       description: 'A selection dropdown can allow multiple selections.',
     },
+    maxSelections: {
+      type: Number,
+      default: Infinity,
+      description: 'Maximum possible selections when using multiple selection',
+    },
```

Comment:
Not 100% sure about this. It isn't a functionality that Semantic UI provides, we shouldn't provide it either. Also, once you add one validation, you will be expected to add all kind of validations to your input fields.
We could consider a validation directive as a separate task. What are your thoughts?

Reply:
Sorry, I didn't see that